src/components/Visualization/Visualization.js

```diff
@@ -153,6 +154,13 @@ export const Visualization = ({
     }
 
     const formatCellHeader = (header) => {
+        let headerName = header.column
+        let dimensionId = header.name
+
+        const match = Object.entries(headersMap).find(
```

Comment:
Im a bit confused about this use of `Object.entries`, since it looks like only `match[0]` (key) is used below. 
If you specifically only need the key in the end, then you might as well do `Object.keys(headersMap).find(key => headersMap[key].name === header.name)`.